### DISCUSSION TITLE
Minting IRIs

### DISCUSSION THREAD

**Discussion by CECSpecialistI** [crystalyragui/MARC2RDA#450] [General]
We should decide how and when to mint IRIs! We have been talking about it during meetings and this will be a central place to link to notes, starting with our [Minting IRIs Brain Splat](https://docs.google.com/document/d/1I81ICKfBNgG5EWznR8vaAosOwtSchjStmFAZsGM_IAc/edit?usp=drive_link). Discussion should also continue during meetings, as much more remains to be decided.

[Designing IRIs Discussion](https://github.com/uwlib-cams/MARC2RDA/discussions/472)

### Here are questions we need to answer related to minting IRIs

******* Please reply to each question separately, using "Write a reply" instead of starting a new comment. *******

**Comment by cspayne** [comment]
### Question: Are we in agreement that for a 720 field “Added Entry – Uncontrolled Name”, we do not mint an IRI for the agent? 

As Gordon said in [Access Point Questions](https://github.com/uwlib-cams/MARC2RDA/discussions/453#discussioncomment-9828929): 

> We are only minting a related entity when a) we can attach an appellation for conformance with RDA and b) when we can extract additional descriptive data from the record and attach it. If these conditions are not met, we should not mint a related entity and just use a literal for the record's data value.

720 does not provide us with any additional information that we could attach to the Agent entity, so instead the object of the property would be a string value based on subfield $a.  

So ```720 | 1# $aBlacklock,   Joseph``` would become:
```ttl
<manifestationIRI> rdamd:P30268 “Blacklock, Joseph” . //has related person of manifestation
```

**Reply by tmqdeborah** [reply]
I agree

**Comment by cspayne** [comment]
### Question: When there is a $1 value from an approved source should we use this value as the Entity IRI instead of minting an IRI? 

An approved source would be a source that the group has agreed provides RWO IRIs. 

#### And: If we are using $1 as the Entity IRI, can we add additional triples describing that entity such as the inverse relationship?

**Reply by GordonDunsire** [reply]
The derefenced description set from a LoC rwo for a Person entity uses rdf:label for what we are assuming is the authorized access point. The dereferenced description set includes the component values of the aap as RDA-compatible elements (I think).

For example, for Jane Austen:

```ttl
<http://id.loc.gov/rwo/agents/n79032879> rdfs:label "Austen, Jane, 1775-1817"> .
<http://id.loc.gov/rwo/agents/n79032879> isA <http://id.loc.gov/ontologies/bibframe/Person> .
<http://id.loc.gov/rwo/agents/n79032879> isA <http://xmlns.com/foaf/0.1/Person> .
<http://id.loc.gov/rwo/agents/n79032879> madsrdf:birthDate "1775-12-16"^^<http://id.loc.gov/datatypes/edtf/EDTF> .
<http://id.loc.gov/rwo/agents/n79032879> madsrdf:deathDate "1817-07-18"^^<http://id.loc.gov/datatypes/edtf/EDTF> .
```

There also statements that cannot be derived from the aap or other data recorded in a MARC 21 record that is to be transformed.

This suggests that it would be worthwhile to process the heading data as if the transform has minted an IRI. That is, add the aap as an RDA element, mint a nomen for the aap, and give its scheme as LCNAF:

```ttl
<http://id.loc.gov/rwo/agents/n79032879> rdaao:P50411 <nomenAustenJane1775-1717> . // has authorized access point for person
<nomenAustenJane1775-1717> rdan:P80068 "Austen, Jane, 1775-1817" . // has nomen string
<nomenAustenJane1775-1717> rdan:P80069 <http://id.loc.gov/authorities/names>.  // has scheme of nomen
```

I don't think it's useful to add descriptive statements that are parsed out of the heading because they are likely to be covered by the dereferenced data.

The 'non-opaque' nomen IRI is necessary to automatically de-duplicate minted nomens and their statements, as with subjects.

**Reply by cspayne** [reply]
This makes sense to me, and I am in agreement that we need to use 'non-opaque' nomen IRIs in order to automatically de-duplicate nomens.

**Reply by tmqdeborah** [reply]
@GordonDunsire, just to be sure that I’m understanding your suggestion correctly, I'm going to rephrase it slightly; we will:
- record an external IRI in a subfield $1 from a headings field as the range of a triple, e.g.:
  - WorkIRI _has author person_ <http://id.loc.gov/rwo/agents/n79032879> . // external IRI
- use the AAP from the headings field to mint an IRI for a nomen, e.g.:
  -  nomenAustenJane1775-1817  . // from 100 1  $a Austen, Jane, $d 1775-1817
- create another triple that has the external IRI as its domain and the NomenIRI as its range, e.g.:
  - <http://id.loc.gov/rwo/agents/n79032879> _has authorized access point for person_ nomenAustenJane1775-1817
- create triples with the NomenIRI as their domain, e.g.:
  - nomenAustenJane1775-1817 _has nomen string_ "Austen, Jane, 1775-1817"
  - nomenAustenJane1775-1817 _has scheme of nomen_ <http://id.loc.gov/authorities/names>

Please confirm that this also means that we can also create a triple for an inverse relationship, e.g.:
- <http://id.loc.gov/rwo/agents/n79032879> _is author person of work_ WorkIRI

**Reply by GordonDunsire** [reply]
@tmqdeborah: I do not recommend creating inverse triples from external datasets or value vocabularies. This is a choice for the agents who maintain the external data or for applications, and it is considered unethical in some linked data communities. Without full provenance, a user does not know if, in this case, the triple is 'official' LoC data.

**Reply by GordonDunsire** [reply]
@cspayne: Terminology needs to be clearer:

- We will record an external IRI in a subfield $1 from a headings field as the object of a triple with a relationship property. Range is associated with the RDA relationship properties; the range essentially types the object of the triple, so in this case it types the IRI (what class of entity the IRI represents).
- We will not make statements about the IRI in subfield $1 except for adding an authorized access point triple with the heading as the object. If the source of the heading is given, the heading object will be a minted nomen; if not, the object will be a literal string. If a nomen is minted, we will make statements about the nomen that include the nomen string and the scheme/source of the nomen.
- We will develop a preference list of subfield $1 sources if the field contains multiple $1 values.

Apologies for being picky, but our discussions occasionally get confused over what is being discussed :-)

**Reply by cspayne** [reply]
@GordonDunsire 
Thank you for making these clearer! I think you put it into words much better than I am capable of :)

**Comment by cspayne** [comment]
### Question: What $0 sources do we want to approve? 

We previously decided that when there is a $0 or $2, we will mint an Agent Entity (and Nomen Entity) and use an IRI based on the authorized access point to allow for automatic de-duplication. 

We will need to develop a list of approved sources for $0s, which will help determine the 'source' part of the IRIs we mint (i.e. http://marc2rda.edu/agent/source/accesspoint) and the scheme of nomen value. 

What $0 sources do we want to approve? Is this something that should be done as a group through a discussion/meeting or by one person and reviewed?

**Reply by GordonDunsire** [reply]
@cspayne: To clarify, I think we discussed transforming an 'approved' IRI in subfield $0 to an existing or potential rwo IRI that has not been recorded in subfield $1.

@AdamSchiff pointed out that there is a regular pattern in LCNAF that distinguishes the 'authority' recorded in subfield $0 and the rwo recorded in subfield $1, so the transform can use this pattern to create an LoC rwo IRI. The transform can assume that the LoC rwo IRI exists, or should exist.

I showed a similar approach in the subject heading and classification document for FAST identifiers recorded in $0 of 6XX fields; these can be transformed into FAST IRIs that exist or should exist.

I don't recall any other cases that are safe enough for this to work, but there may be suitable RDA or cloned RDA IRIs lurking in subfield $0 instead of subfield $1. For the moment, the list of approved sources has only 2 entries :-)

**Reply by cspayne** [reply]
Reply from Adam via email: 

There are a few others that have both authority URIs and RWO URIs: GettyULAN and GettyTGN, and NDL Web Authorities.  These are the ones that show both $0 and $1 URIs in the table at https://www.wikidata.org/wiki/Wikidata:WikiProject_URIs_in_MARC.

Adam L. Schiff
Principal Cataloger
University of Washington Libraries
(206) 543-8409
***@***.***

**Reply by cspayne** [reply]
@GordonDunsire @AdamSchiff 
What about viaf IRIs? I've found 700s with them in $0.

**Reply by GordonDunsire** [reply]
@cspayne: VIAF describes itself as a 'name authority service' and as such it is encoded correctly as subfield $0. The IRI dereferences to a skos:Concept (with a different IRI) which wraps around (via foaf:focus) the rwo represented by the IRI. An RDA linked data application can discard the wrapper, so the IRI is valid as a rwo value for an entity-agent relationship. However, the rwo is typed by VIAF as schema.Person which includes legendary and fictitious 'persons'. RDA Person has a narrower definition and would be mapped as an entity subtype of schema:Person. This is not a problem if an RDA relationship property is used: the range of RDA Person is programatically equivalent to schema:Person. But the association between RDA and schema is asymmetric, and we cannot assume that schema:Person is programatically equivalent to RDA Person. In other words, a VIAF agent IRI is valid as an instance of RDA agent and its subtypes if it selected by a human agent carrying out original RDA cataloguing, but it is not safe for the transform unless there is other evidence that the IRI represents an RDA agent or subtype (real, etc.).

The same analysis should be carried out for the sources in @AdamSchiff' s table. See the discussion on Modeling Agents and Personas/Pseudonyms (https://github.com/uwlib-cams/MARC2RDA/discussions/340). There is a generic issue with legacy metadata because AACR2 and other standards did not distinguish rwo persons from conceptual persons, a practice which persists in current standards.

**Reply by cspayne** [reply]
@GordonDunsire @tmqdeborah
If we cannot programmatically determine whether an IRI represents an RDA agent or not, is it safe to even be minting agents for 600, 610, and 611 and thus asserting that these are access points for RDA agents? Couldn't these be fictional people?

**Reply by cspayne** [reply]
@GordonDunsire @tmqdeborah @AdamSchiff 

Additonally, for $0s that we cannot transform to RWO IRIs, my understanding was that we could still use these to say that this entity has an authorized access point from [source derived from $0]. 
This would mean that when we mint an IRI for that entity, the IRI would include some indication of that source + the access point so that the IRI could be automatically de-duplicated.
So for:

`100	1#$aParagas, Erickson Mesina, $d1984- $eauthor. $0(orcid)0000-0001-5292-9223`

We could say

```ttl
<workIRI> [has author person] <http://marc2rda.edu/agent/orcid/paragas%2Cericksonmesina%2C1984->
<http://marc2rda.edu/agent/orcid/paragas%2Cericksonmesina%2C1984-> [has authorized access point for person] <http://marc2rda.edu/fake/nom/orcid/paragas%2Cericksonmesina%2C1984-> 
<http://marc2rda.edu/fake/nom/orcid/paragas%2Cericksonmesina%2C1984-> [has nomen string] "Paragas, Erickson Mesina, 1984-"
<http://marc2rda.edu/fake/nom/orcid/paragas%2Cericksonmesina%2C1984-> has scheme of nomen <http://id.loc.gov/vocabulary/identifiers/orcid>
```

Can we say that though? Is the source of the identifier also the source of the access point?

**Reply by cspayne** [reply]
Adam's response: 

This would not be correct.  Firstly, my understanding is that ORCID IDs go in $1, not $0.  Secondly, ORCID names are displayed in direct order and never include dates, so ORCID would never be the source of an access point in $a, $d, etc.   The source should be the LC/NACO Authority File, but it is not infrequent that a cataloger (especially one working at a non-NACO library) formulates the access point according to RDA (or earlier, AACR2) but there is no authority record for it.  They could point to the ORCID for the person, but that should be put in $1.

Adam

Adam L. Schiff
Principal Cataloger
University of Washington Libraries
(206) 543-8409
***@***.***

**Reply by pennylenger** [reply]
> @cspayne: VIAF describes itself as a 'name authority service' and as such it is encoded correctly as subfield $0. The IRI dereferences to a skos:Concept (with a different IRI) which wraps around (via foaf:focus) the rwo represented by the IRI. An RDA linked data application can discard the wrapper, so the IRI is valid as a rwo value for an entity-agent relationship. However, the rwo is typed by VIAF as schema.Person which includes legendary and fictitious 'persons'. RDA Person has a narrower definition and would be mapped as an entity subtype of schema:Person. This is not a problem if an RDA relationship property is used: the range of RDA Person is programatically equivalent to schema:Person. But the association between RDA and schema is asymmetric, and we cannot assume that schema:Person is programatically equivalent to RDA Person. In other words, a VIAF agent IRI is valid as an instance of RDA agent and its subtypes if it selected by a human agent carrying out original RDA cataloguing, but it is not safe for the transform unless there is other evidence that the IRI represents an RDA agent or subtype (real, etc.).
> 
> The same analysis should be carried out for the sources in @AdamSchiff' s table. See the discussion on Modeling Agents and Personas/Pseudonyms (#340). There is a generic issue with legacy metadata because AACR2 and other standards did not distinguish rwo persons from conceptual persons, a practice which persists in current standards.

@GordonDunsire Hi Gordon, I've studied your comparison of VIAF and RDA, and got some questions. Besides schema:Person, there are also types like schema:CreativeWork, schema:Organization, schema:Place, etc. Should I compare the definition of these types with their corresponding RDA entities such as Work, Corporate Body, and Place? Additionally, how should I handle entities like schema:Event and schema:Action, which don't have direct counterparts in RDA?
I also found that other sources, like BBC Thing, include core concepts such as People, Places, Events, Organizations, and Themes. Is the task to check $0 semantics to compare these definitions of these types with RDA entities? And I'm struggling to distinguish between authority  and real-world object. Do you have any good method of identifying whether an entity is an authority or an rwo? Thank you!

**Reply by GordonDunsire** [reply]
@cspayne"
 
`100	1#$aParagas, Erickson Mesina, $d1984- $eauthor. $0(orcid)0000-0001-5292-9223`

should generate:

```ttl
<workIRI> [has author person] <http://marc2rda.edu/agent/lcnaf/paragasericksonmesina1984-> . // remove all punctuation except hyphen; we assume lcnaf is the source of the heading, not orcid
<http://marc2rda.edu/agent/lcnaf/paragasericksonmesina1984-> [has authorized access point for person] "Paragas, Erickson Mesina, 1984-" . // mapped from $a, &d - use aap as string, not nomen because no source is given for this string
<http://marc2rda.edu/agent/lcnaf/paragasericksonmesina1984-> [has identifier for person] '(orcid)0000-0001-5292-9223' . // mapped from $0
<http://marc2rda.edu/agent/lcnaf/paragasericksonmesina1984-> [has identifier for person] 'https://orcid.org/0000-0001-5292-9223' . // transformed from $0 with standard pattern from table
```

`100	1#$aParagas, Erickson Mesina, $d1984- $eauthor. $1https://orcid.org/0000-0001-5292-9223`

should generate the same statements because the ORCID IRI has type schema:Person which covers fictitious and 'undead' persons. So in general the transform cannot assume that an ORCID IRI references a real person. We can, however, assume that it is an identifier for a real person in fields 100 and 700 because the RDA interpretation is that it identifies a persona or pseudonym of a real person and therefore identifies the person.

@pennylenger: 

Yes, please compare the RDA entities with similar schema entities. More generally, compare an RDA entity with any similar entities from other sources. For example, foaf:Person is also defined as including imaginary, etc. persons; in fact, schema:Person and foaf:Person have near-identical definitions and are supertypes of rda:Person. You may find that other entities such as place also include fictitious or legendary places, which means they are supertypes of the rda:Place entity and their IRIs can only be used as identifiers. On the other hand, wikidata:Q5 (human) is directly equivalent to rda:Person because it excludes imaginary, etc. 'persons'.

It is sufficient to relate the RDA entity to the source entity as equivalent (ok to use), supertype (not ok to use), and subtype (ok to use). Hypothetical examples of the last (subtype) are 'man', 'woman', 'child', etc.

All 'authorities' are rwos; the question is: what type of entity does the URL/IRI reference. An 'authority' maps to an instance of RDA manifestation because it is a document, even if the document is encoded in html or is an rdf graph (rdf/xml, etc.). Examine the statements with the authority IRI as subject in the rdf graph. The statements will include predicates that are associated with, say, an instance of person, but may include predicates that are associated with the description of that person (such as who created the description) or administrative data associated with the 'authority record' such as date entered into the system.

If all of the statements in the rdf graph have predicates directly related to the person, it is ok to use the authority IRI as a person IRI. But if any of the statements use predicates about data provenance, record administration, etc. then we assume the authority IRI references a manifestation.

We have confirmed in other discussions that this can be modeled in RDA, but first we need to know what class/entity/type the IRI references, including rda:Manifestation for 'authorities', schema:Person, foaf:Person, etc. Then we need to know if the class/entity/type is equivalent to, a supertype of, or a subtype of the RDA entity. After that, we can discuss in detail the treatment of subfield $0.

If the source has no definition of its entity, we must discard the IRIs from the source.

If the source definition is difficult to compare with RDA, post both definitions to this thread and we can make a decision.

**Reply by pennylenger** [reply]
wikidata:Q5: any member of Homo sapiens, unique extant species of the genus Homo, from embryo to adult

**Reply by pennylenger** [reply]
Wikidata person (Q215627) being that has certain capacities or attributes constituting personhood (avoid use with P31; use Q5 for humans) . Wikidata person seems more equivalent to RDA:person.

**Reply by pennylenger** [reply]
Because RDA: person is an agent and an agent is who is capable of deliberate actions, of being granted rights, and of being held accountable for its actions."

**Reply by GordonDunsire** [reply]
@pennylenger: The RDA definition of person is 'An agent who is an individual human being who lives or is assumed to have lived', which is an equivalent match to wikidata Q5. Wikidata person is a reflection of pre-LRM library practice of including personas, legendary human-like entities, fictional persons, etc. as persons. It is equivalent to foaf Person, ISNI person, etc. All of these ontologies are rooted in traditional library practices based on strings for entity identification that are derived from 'name authority' files. Originally, the sole sources of names were those that appeared in manifestation statements of responsibility (which may be fake, erroneous, or misleading).

In the real world, fictional persons, etc. do not actually create works, etc.; their names are used as pseudonyms of a human. Animal performers are excluded from RDA Agent for similar reasons, no matter how cute they are :-)

**Reply by CECSpecialistI** [reply]
Wikidata human (Q5) is equivalent to RDA person. Wikidata person is not equivalent to any RDA class.

**Comment by AdamSchiff** [comment]
Case in point:

600 10  Holmes, Sherlock.

For personal names without $c (Fictitious character) or some other qualifier ($c Legendary character, $c Greek deity, $c Vampire, $c Race horse, etc.) that indicates fictitiousness, you'd have to examine the NAR to see if there are other indications that the entity is non-human.

Adam L. Schiff
Principal Cataloger
University of Washington Libraries
(206) 543-8409
***@***.***
________________________________

**Reply by tmqdeborah** [reply]
Doesn't this question also apply to other X00, X10, X11 fields?

For example,  
010__ |a  2015506441 
1001_ |a Kirk, James T. |c (Fictitious character), |d 2233-2371, |e author. 

Although we can use the qualifier "(Fictitious character)" and other such qualifiers to prevent recording them as related agents:
- what do we do with these rejected headings and where can we find a (complete?) list of other qualifiers;
- and, as Adam indicates, there will be some headings that have not had these qualifiers added
- and checking the NAR would mean checking every heading, and, even then, there might not be an NAR for the heading.

I think we might have to check for whatever qualifiers we can identify and then decide what to do with the headings we find and let the ones we cannot find go through as errors. 

But what do we do with those headings that we identify as fictitious? Perhaps just treat the name as a pseudonym? 

We have a [discussion page for pseudonyms](https://github.com/uwlib-cams/MARC2RDA/discussions/340), but have we given Cypress explicit instructions about how to transform them?

**Reply by GordonDunsire** [reply]
I don't think it's feasible for the transform to lookup any authority files; it would take too long, plus the matching conditions will never get to sufficient levels of accuracy.

There is still some confusion between entity/agent and nomen. We have not yet discussed the minting of entities other than nomen and skos:Concept; the discussion in this thread so far has focused on obtaining an IRI for the entity from an external dataset. There are several issues that should make us cautious about the aap de-deduplication method, including undifferentiated headings and fictitious/legendary headings. RDA says such headings are appellations of human agents; the problem lies with identifying which agents.

The discussion on pseudonyms assumes that the rwo agent is known and identified, but I don't see how the transform can obtain this information during processing. If there is no information about the rwo and we cannot accurately transform subfield $0 to $1 or subfield $0 is missing, then we can only mint a running-number/opaque IRI for the agent. If there is no source for the heading, we don't mint an IRI for the nomen, and we use the access point element with the heading string.

'''
<workIRI> [has author person] <http://marc2rda.edu/agent/transform/nnnn> .
<http://marc2rda.edu/agent/transform/nnnn> [has access point for person] "Kirk, James T. (Fictitious character), 2233-2371" .
'''

If a source for the heading is known, but not the IRI for the agent as rwo:

'''
<workIRI> [has author person] <http://marc2rda.edu/agent/transform/nnnn> .
<http://marc2rda.edu/agent/transform/nnnn> [has authorized access point for person] <http://marc2rda.edu/agent/source/KirkJamesTFictitiouscharacter2233-2371>
<http://marc2rda.edu/agent/source/KirkJamesTFictitiouscharacter2233-2371> [has nomen string] "Kirk, James T. (Fictitious character), 2233-2371" .
<http://marc2rda.edu/agent/source/KirkJamesTFictitiouscharacter2233-2371> [has scheme of nomen] <schemeIRI> .
'''

This is risky because there is automatic de-duplication of the nomen, even if it is used by two different agents. Risk assessment is dependent on the source of the heading ... which is another reason why we need to know more about the sources of entity identifiers.

**Reply by cspayne** [reply]
Why would we do automatic de-duplication of the nomen if we were not doing automatic de-duplication for the agent? 
By using both the source in $2 and the aap to mint an IRI aren't we minimizing the risk of accidentally equating different agents?

**Reply by GordonDunsire** [reply]
@cspayne: Automatic de-duplication of a nomen is only feasible when we know that the nomen(s) are appellations of the same entity. We are minting nomens for headings only when we have an external IRI for the entity, to add the finer granularity of access point or authorized access point to the entity IRI. The transform will do this every time it encounters the heading + external IRI combination and output multiple copies of the same triple. If we don't have an external IRI for the entity and mint one, we can't safely de-duplicate using the heading. For example, the pseudonym 'Sherlock Holmes' has been used by multiple different agents. Instead, it is safer to mint opaque IRIs for each occurence and leave de-duplication for post-transform processing.

**Reply by cspayne** [reply]
@GordonDunsire 
That makes sense, thank you. When you say we can't safely de-duplicate using the heading when we mint an IRI, is this because one source may use the same access point for multiple agents?
So we would mint opaque IRIs for these agents, but they may share the same nomen?

**Reply by GordonDunsire** [reply]
@cspayne: One source + same access point = undifferentiated access point. An entity cannot share the 'same' nomen because a nomen is distinguished by both its nomen string and the entity of which it is an appellation. We therefore cannot mint a heading-based nomen unless we are fairly certain that it is the appellation of a one/unique instance of an entity.

**Reply by cspayne** [reply]
So do we want to try and do any de-duplication for agents and their nomens when we mint IRIs? Or is the best option to use opaque IRIs every time? Can we come up with a list of sources that we can be fairly certain the appellation is unique and we want to mint heading-based IRIs for?

**Reply by CECSpecialistI** [reply]
I think deduplication needs to be saved for phase II. We can do pre-processing to try and add as many $1's as possible to avoid having agents/nomens without them as much as we can. But same access point isn't always same entity (even though it's theoretically supposed to be). It's easier to merge duplicates than it is to disentangle bad matches.

**Reply by GordonDunsire** [reply]
@cspayne, @CECSpecialistI: In another discussion I have suggested that we try the de-duplication for all 'heading' entities, but not for any minted nomens (as explained above). That's why I say:

```ttl
<workIRI> [has author person] <http://marc2rda.edu/agent/lcnaf/paragasericksonmesina1984-> . // remove all punctuation except hyphen 
```

Note that I'm being sloppy with the minted IRI bases; apologies.

**Comment by AdamSchiff** [comment]
Correctly formed VIAF IRIs should be in $1.  VIAF doesn’t have a single authorized access point, so it isn’t appropriate in $0.  We should either be able to just use the VIAF IRI as is if it is correctly formed, or take the identifier part (the numeric ID) and turn it into a correctly formed IRI.

Example:
VIAF ID: 21647077
IRI: http://viaf.org/viaf/21647077

So if $0 has
http://viaf.org/viaf/21647077
 it should be treated as $1

If it has
https://viaf.org/viaf/21647077
 or
https://viaf.org/viaf/21647077/<https://viaf.org/viaf/21647077/#Dunsire,_Gordon.>
It should be converted to http://viaf.org/viaf/21647077

If it has the VIAF IRI for the LC authority record in $0, https://viaf.org/processed/LC%7Cnb2001072552, we could take ID part and plug it into the LC RWO agent IRI:
http://id.loc.gov/rwo/agents/nb2001072552

Adam

Adam L. Schiff
Principal Cataloger
University of Washington Libraries
Box 352900
Seattle, WA 98195-2900
aschiff @ uw.edu
________________________________

**Comment by AdamSchiff** [comment]
There won’t be a $2 in access point and subject fields for agents constructed according to RDA (or AACR2 or AACR). The assumption in 1XX, 7XX, and 600-611 with second indicator 0 is that the agents are either in the LC/NACO authority file in the same form or that they are constructed in accordance with RDA/AACR2/AACR. This isn’t always true, but that’s the basic assumption.

Adam L. Schiff
Principal Cataloger
University of Washington Libraries
Box 352900
Seattle, WA 98195-2900
aschiff @ uw.edu
________________________________

**Reply by lake44me** [reply]
That's what I was always taught -- but my library followed AACR2 when I started a long time ago, and LC Rule Interpretations (before there was a PCC, probably).  In our copy cataloging, we manually checked NAF and "updated headings" in the old days... then we used an authority control vendor and automated matching, which probably was successful most of the time.  That may not have been true for others who contributed records we used, especially if those came from countries where English was not a primary language.  

It's not explicitly stated that 1xx names are "controlled".  An assumption might be made that this is so for 700, 710, and 711 names, since 720 is for an "uncontrolled name".  730, a "uniform title" is certainly controlled by a certain set of rules, but what version would not be clear for either "aacr" or "rda".  

A possible clue might be the institution symbols in the 040, in addition to 040  $e .  If the record has been edited by a library that follows use of NACO Authority File to control access points (e.g. DLC) it is likely that the access points follow the rules in $e, even if there is no authority record whose authorized access point matches the form in the field.   Relying on symbols of other institutions (Harvard?  U. of Wash.?  Emory?) would be going too far I think.  

Perhaps we could make note of our choice of "access point" versus "authorized access point" in absence of data from $0 , $1, or $2 source, in notes we provide to implementers, and that that choice could be adjusted in the transformation tools by an institution that was only transforming its own MARC records and was clear about its historical cataloging practices for these fields.

This is why I think it important to have at least a small sample of records from many other sources in addition to the Library of Congress record set to demonstrate results of our transformations.  Don't know how/whether we can get them without running afoul of OCLC's record "ownership" though.

**Reply by AdamSchiff** [reply]
RDA records will have 040 $e rda, but AACR2 records don't usually use 040 $e.  Instead the rules are found in Leader/18:  
Descriptive cataloging form
[blank] - Non-ISBD                   
a - AACR 2                   
c - ISBD punctuation omitted 
i - ISBD punctuation included                   
n - Non-ISBD punctuation omitted                   
u - Unknown

**Comment by CECSpecialistI** [comment]
@gerontakos Now would be a fun time for you to pop out from LC with a brilliant solution for this problem!
